x-pack/plugins/watcher/server/models/action_status_model/action_status_model.test.js

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+
+import { ACTION_STATES } from '../../../common/constants';
+import { ActionStatusModel } from './action_status_model';
+
+describe('ActionStatusModel', () => {
```

Comment:
This diff looks significant, but all that changed was:

* Rename of `ActionStatus` -> `ActionStatusModel`
* Removal of an unnecessary root-level `describe` block